tests/integration/conftest.py

```diff
@@ -64,10 +64,22 @@ def clean_db():
     before all tests the database is cleared
     """
     with app.app_context():
+        # The table contributor should never be emptied as referenced in real_time_update and trip_update
         tables = [six.text_type(table) for table in db.metadata.sorted_tables]
         db.session.execute("TRUNCATE {} CASCADE;".format(", ".join(tables)))
         db.session.commit()
 
+        # Add two contributors in the table
+        db.session.execute(
+            "INSERT INTO contributor VALUES('realtime.cots','sncf',"
```

Comment:
It's a lot of changes in test files, but it would be better to avoid in tests those values that are in default config.
That way we will see if we get some hardcoded ones from real code (bad) or the ones from tests (good)
```suggestion
            "INSERT INTO contributor VALUES('rt.tchoutchou','sncf',"
```